### ISSUE TITLE
For consistency, eraser icon should only clear canvas

### ISSUE THREAD

**Issue by icarito** [sugarlabs/Pippy#44]
It's a really scary functionality.

**Comment by quozl** [comment]
Ctrl+Z undo works to reverse any eraser button click.  It is also in the scissors menu.  Isn't that discoverable enough?

**Comment by icarito** [comment]
It's the first time I notice this functionality,

I never thought of clearing the editor as a common enough function to
require a toolbar button.

In the place of the user I'd be surprised. Undo is under a sub-palette,
perhaps we could flatten the whole thing, or put eraser into edit
sub-palette. Just a suggestion for discussion.

Thanks!

**Comment by quozl** [comment]
I'm used to it, and it has been there for more than three years, so chances are others are used to it and may have made teaching plans that mention it.  So I'm unwilling to change it myself without some good reason.  If there is room for bringing undo and redo into the main toolbar box, I'm in favour.

It is easy to clear the editor by other means; select all text and type.

**Comment by icarito** [comment]
I understand, thanks!

**Comment by icarito** [comment]
Actually, I'm making a JS version of this activity and need to decide whether to replicate this functionality.
Looking at Turtle Blocks JS, the same icon clears _the canvas_ not the program. This seems more logical to me. @walterbender what do you think, it should be one or the other, no?

**Comment by icarito** [comment]
Okay I figured out where this is coming from. Today I ran an old image with Sugar 0.79, including old Pippy.
In older versions, python examples are featured on a sidebar to the left, and there is a run and erase button to the right.

My feeling is that Pippy was meant as a showcase of Python examples and it could be useful to "clear" the modified example.

Still, for continuity with Turtle, I propose that this button clears the canvas (if wanted we can add a trash bin for clearing the code area).

I will prepare a patch.
